tests/unit/RequestBuilder/RecommendationRequestBuilderTest.php

```diff
@@ -147,4 +156,12 @@ public function shouldFailOnIncorrectSequenceOfUsersWhenMerging(): void
         $builder->setInteraction($interactionCommand);
         $builder->build();
     }
+
+    public function correctSequenceOfUsersProvider()
+    {
+        return [
+          ['test-user-a', 'test-user-a', 'test-user-b', 'test-user-b'], // (A, A -> B, B)
```

Comment:
The comment could instead be used as array key, its then printed in PHPUnit output, like:

```php
'(A, A -> B, B)' => ['test-user-a', 'test-user-a', 'test-user-b', 'test-user-b'],
```

Reply:
done